src/propsheet/precomp.h

```diff
@@ -16,6 +16,8 @@
 #define ISOLATION_AWARE_ENABLED 1
 // -- END WARNING
 
+#define NOMINMAX

```

Comment:
I also added NOMINMAX when fixing #4152. Maybe a rebase would be good.

Reply:
Ooh ok. Thanks.